src/advanced_agent/config/loader.py

```
@@ -0,0 +1,314 @@
+            config_file: 設定ファイルパス（指定しない場合は自動検出）
+            env_prefix: 環境変数プレフィックス
+            validate: 設定検証フラグ
+            
+        Returns:
+            AgentConfig: エージェント設定
+        """
+        try:
+            # 1. 環境変数から設定読み込み
+            config_dict = self._load_from_env(env_prefix)
+            
+            # 2. 設定ファイルから読み込み（環境変数を上書き）
```

Ruff (0.12.2)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)